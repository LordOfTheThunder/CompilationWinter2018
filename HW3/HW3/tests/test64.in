void TestIfElse() {
	if (true) {
		int x = 3;
	}
	else int y = 5;
	y = 3;
}
