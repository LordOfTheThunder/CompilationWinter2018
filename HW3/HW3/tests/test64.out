---end scope---
x INT 0
---end scope---
y INT 0
---end scope---
line 6: variable y is not defined
